<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>797</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Master</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1172</width>
             <height>553</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTabWidget" name="tabWidget_2">
              <property name="tabPosition">
               <enum>QTabWidget::North</enum>
              </property>
              <property name="tabShape">
               <enum>QTabWidget::Rounded</enum>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionNew_Project"/>
    <addaction name="actionNew_Entity_Type"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>19</width>
     <height>18</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTerminal"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/document-new-6.png</normaloff>:/icons/document-new-6.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionNew_Entity_Type">
   <property name="text">
    <string>New Entity Type</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/document-open-7.png</normaloff>:/icons/document-open-7.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/utilities-terminal-6.png</normaloff>:/icons/utilities-terminal-6.png</iconset>
   </property>
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/document-save-as-4.png</normaloff>:/icons/document-save-as-4.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/mouse_pointer.png</normaloff>:/icons/mouse_pointer.png</iconset>
   </property>
   <property name="text">
    <string>Selection</string>
   </property>
  </action>
  <action name="actionLink">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/insert-link-2.png</normaloff>:/icons/insert-link-2.png</iconset>
   </property>
   <property name="text">
    <string>Link</string>
   </property>
  </action>
  <action name="actionAutoLink">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/insert-link.png</normaloff>:/icons/insert-link.png</iconset>
   </property>
   <property name="text">
    <string>AutoLink</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/osa_lightning.png</normaloff>:/icons/osa_lightning.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionAutoZoom">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/star.png</normaloff>:/icons/star.png</iconset>
   </property>
   <property name="text">
    <string>AutoZoom</string>
   </property>
  </action>
  <action name="actionDotLayout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/dotlayout.png</normaloff>:/icons/dotlayout.png</iconset>
   </property>
   <property name="text">
    <string>DotLayout</string>
   </property>
  </action>
  <action name="actionNeatoLayout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/osa_lifecycle.png</normaloff>:/icons/osa_lifecycle.png</iconset>
   </property>
   <property name="text">
    <string>NeatoLayout</string>
   </property>
  </action>
  <action name="actionFdpLayout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/osa_lifecycle.png</normaloff>:/icons/osa_lifecycle.png</iconset>
   </property>
   <property name="text">
    <string>FdpLayout</string>
   </property>
  </action>
  <action name="actionSfdpLayout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/osa_lifecycle.png</normaloff>:/icons/osa_lifecycle.png</iconset>
   </property>
   <property name="text">
    <string>SfdpLayout</string>
   </property>
  </action>
  <action name="actionElastic">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/osa_lifecycle.png</normaloff>:/icons/osa_lifecycle.png</iconset>
   </property>
   <property name="text">
    <string>ElasticLayout</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
